src/synapse/azext_synapse/_help.py

```diff
@@ -279,96 +331,95 @@
+        --spark-pool-name testsparkpool --executor-size Small --executors 4
 """
 
 helps['synapse spark session list'] = """
 type: command
-short-summary: List all spark session jobs under the specific spark pool.
+short-summary: List all Spark sessions.
 examples:
-  - name: List all spark session jobs under the specific spark pool.
+  - name: List all Spark sessions.
     text: |-
         az synapse spark session list --workspace-name testsynapseworkspace --spark-pool-name testsparkpool
 """
 
 helps['synapse spark session show'] = """
 type: command
-short-summary: Get a specific spark session job under the specific spark pool.
-long-summary: Get the spark session job under the specific spark pool with session id.
+short-summary: Get a specific Spark session.
 examples:
-  - name: Get a spark session job under the specific spark pool with session id.
+  - name: Get a Spark session.
     text: |-
-        az synapse spark session show --id 1 --workspace-name testsynapseworkspace --spark-pool-name testsparkpool
+        az synapse spark session show --livy-id 1 --workspace-name testsynapseworkspace --spark-pool-name testsparkpool
 """
 
 helps['synapse spark session cancel'] = """
 type: command
-short-summary: Cancel a specific spark session job under the specific spark pool.
-long-summary: Cancel the spark session job under the specific spark pool with session id.
+short-summary: Cancel a specific Spark session.
 examples:
-  - name: Cancel a spark session job under the specific spark pool with session id.
+  - name: Cancel a spark session.
     text: |-
-        az synapse spark session cancel  --id 1 --workspace-name testsynapseworkspace --spark-pool-name testsparkpool
+        az synapse spark session cancel  --livy-id 1 --workspace-name testsynapseworkspace --spark-pool-name testsparkpool
 """
 
 helps['synapse spark session reset-timeout'] = """
 type: command
-short-summary: Reset the spark session timeout time.
-long-summary: Reset the spark session timeout time under the specific spark pool with session id.
+short-summary: Reset the Spark session timeout time.
 examples:
-  - name: Reset the spark session timeout time.
+  - name: Reset the Spark session timeout time.
     text: |-
-        az synapse spark session reset-timeout --id 1 --workspace-name testsynapseworkspace --spark-pool-name testsparkpool
+        az synapse spark session reset-timeout --livy-id 1 --workspace-name testsynapseworkspace --spark-pool-name testsparkpool
 """
 
-helps['synapse spark session-statement'] = """
+helps['synapse spark statement'] = """
 type: group
-short-summary: Manage Synapse Spark Session Statements.
+short-summary: Manage Synapse Spark statements.
 """
 
-helps['synapse spark session-statement create'] = """
+helps['synapse spark statement create'] = """
 type: command
-short-summary: Submit a spark statement to a spark session.
+short-summary: Submit a Spark statement.

```

Comment:
>Submit [](start = 15, length = 6)

I would suggest verb "invoke"

Reply:
Also for the command group name.

---
In reply to: [418045381](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418045381) [](ancestors = 418045381)

Reply:
fixed. changed it to invoke

---
In reply to: [418047097](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418047097) [](ancestors = 418047097,418045381)